lib/diagnostic.rb

```diff
@@ -93,7 +100,7 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = arr.shift(2).shift(3)
```

Comment:
So shift will actually remove the element from the front of the array. We want to use `arr.pop 2`